lesson08/assignment/video_cli/l8/src/main.js

```diff
@@ -1,13 +1,44 @@
 import Vue from 'vue';
 import App from './App.vue';
 //import router from './router';
+import Home from './Home.vue';
+import Top3 from './top3.vue';
+import ImagePage from './imagepage.vue'
 
 import 'bootstrap';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import VueRouter from 'vue-router';
 
 Vue.config.productionTip = false;
 
+Vue.use(VueRouter);
```

Comment:
This is where your router code should be setup.  The docs and lessons may not have been super clear on this.